third_party/docfx-doclet-143274/src/test/resources/expected-generated-files/toc.yml

```diff
@@ -169,7 +177,8 @@
       status: "beta"
       items:
       - uid: "com.microsoft.samples.google.v1beta"
-        name: "Package summary"
+        name: "Package Overview"
```

Comment:
Not sure we want to change this from 'summary' to 'Overview', but we definitely want to make the second word lowercase. ("overview" or "summary").

Reply:
ah yes this was legacy when I was testing it alongside the old Package summaries. Updated.